src/components/Dialog/Dialog.tsx

```diff
@@ -132,6 +132,10 @@ export interface DialogProps extends VibeComponentProps {
   getDynamicShowDelay?: () => { showDelay: number; preventAnimation: boolean };
   content?: (() => JSX.Element) | JSX.Element;
   children?: ReactElement | ReactElement[] | string;
+  /**
```

Comment:
how complicated is to add test for this ability? (if flag is on and hover trigger pass check if focus trigger the dialog)

Reply:
after removing the flag we will remove the flag from the test also

Reply:
maybe let's also add test that this ability not working when the flag is off

Reply:
will try to write the test 👍